src/control/zoomControl.js

```diff
@@ -0,0 +1,34 @@
+import {
+    Control
+} from './control';
+import {
+    DomEvent
+} from '../util/domEvent';
+export class ZoomControl extends Control {
+    constructor() {
+        const controlContainers = document.getElementById('fe-conrols-container');
+        const ele = document.createElement('div');
+        ele.className = 'fe-control-zoom fe-control';
+        ele.innerHTML = `<button class="fe-zoom-in" type="button" title="Zoom in">+</button>
+            <button class="fe-zoom-out" type="button" title="Zoom out">−</button>`;
+        controlContainers.appendChild(ele);
+        super(ele);
+
+        this.btnZoomIn = ele.getElementsByTagName('button')[0];
```

Comment:
once get dom， better  performs ：
const buttons=ele.getElementsByTagName('button');
this.btnZoomIn=buttons[0];
.......